src/App.jsx

```diff
@@ -14,33 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD__ALPHABET = 'alphabet';
+const SORT__FIELD__LENGTH = 'length';
```

Comment:
The naming of constants `SORT_FIELD__ALPHABET` and `SORT__FIELD__LENGTH` is inconsistent. Consider using a consistent naming convention, such as `SORT_FIELD_ALPHABET` and `SORT_FIELD_LENGTH`, to improve readability and maintainability.